cdap-docs/build.sh

```diff
@@ -120,7 +120,9 @@ function _build_docs() {
   local title=${2}
   display_start_title "${title}"
 
-  clean_targets
+  if [[ ${type} == "docs_set" ]] || [[ ${type} == "docs_web_only" ]]; then
```

Comment:
This exact same conditional appears a couple lines below. Can these not be reordered and `clean_targets` put inside the below conditional?